cassandra/query.py

```diff
@@ -701,6 +720,16 @@ def _maybe_set_routing_attributes(self, statement):
                 self.routing_key = statement.routing_key
                 self.keyspace = statement.keyspace
 
+    def _update_custom_payload(self, statement):
+        if statement.custom_payload:
+            if self.custom_payload is None:
+                self.custom_payload = {}
+            self.custom_payload.update(statement.custom_payload)
```

Comment:
Should this behavior be explicitly enabled? This seems like unexpected behavior to me.